src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 2px 6px 2px 2px;
+  width: 12px;
+  height: 12px;
+  background-image: url(images/star.svg);
+  background-position: cover;
+  display: inline-block;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
```

you can group the selectors

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n+2),
.stars--2 :nth-child(-n+2) {
  background-image: url(images/star-active.svg);
}
```

---

Aight got it